plusminus/plusminus.py

```diff
@@ -255,7 +255,7 @@ def safe_str_mult(a, b):
 
 
 def constrained_factorial(x):
-    """Same as `math.factorial`, but raises `ValueError` if x is under 0 or over 32,767."""
+    """Same as `math.factorial`, but raises `ValueError` if x is under 0 or over 32,768."""
```

Comment:
32, 767 is the correct value for this comment.  "not < 32768" is the same as "over 32, 767".